giddy-banana/app.js

```diff
@@ -0,0 +1,11 @@
+const express = require("express");
```

Comment:
Please use ' import express from "express" ' instead of require, because it's now ES6 standard.